enterprise/server/cmd/action_runner/main.go

```diff
@@ -0,0 +1,662 @@
+package main
```

Comment:
wdyt about calling this whole package "ci_runner" instead of action_runner? Since "action" is already an overloaded term in the bazel world / remote execution space?